examples/node/spec/unit/credential.spec.ts

```diff
@@ -0,0 +1,15 @@
+'use strict';
+import nock from 'nock';
+import Twilio from '../../lib/rest/Twilio';
+
+describe('credential', () => {
+    const twilio = new Twilio();
+
+    it('should update an aws credential', () => {
```

Comment:
Should we also have a delete test?

Reply:
We _could_, I was just focusing on making sure we could do _anything_ against this endpoint.